src/main/java/com/faforever/client/map/MapVaultController.java

```diff
@@ -136,7 +136,7 @@ public void initialize() {
   }
 
   private void searchByQuery(SearchConfig searchConfig) {
-    this.query = searchConfig.getSearchQuery();
+    this.query = searchConfig.getSearchQuery() + ";latestVersion.hidden==\"false\"";
```

Comment:
There are different possible filter dialects, and the field name might change.
Hardcoding this here is, therefore, not a future-proof solution. Also, when we display the search query to the user, this part will be left out so he might wonder why he gets different results.

The proper solution will probably include adding a pre-defined condition as a parameter to `com.faforever.client.vault.search.SearchController#buildQuery`. It's not going to be easy but it will be much more robust.

Reply:
After discussing this with alex, we decided to add the filter in API accessor.